hugolib/site.go

```diff
@@ -976,7 +977,8 @@ func (s *Site) initializeSiteInfo() {
 		sectionPagesMenu:               lang.GetString("sectionPagesMenu"),
 		GoogleAnalytics:                lang.GetString("googleAnalytics"),
 		BuildDrafts:                    s.Cfg.GetBool("buildDrafts"),
-		canonifyURLs:                   s.Cfg.GetBool("canonifyURLs"),
+		CanonifyURLs:                   s.Cfg.GetBool("canonifyURLs"),
```

Comment:
These should not be exported, and this is an indication that `SiteInfo` probably needs an unexported `PathSpec`. Note that `SiteInfo` is what the users get when they say `.Site` from a template.